airbyte-integrations/connectors/destination-langchain/metadata.yaml

```diff
@@ -22,4 +22,9 @@ data:
     sl: 100
     ql: 100
   supportLevel: community
+  breakingChanges:
+    0.1.0:
```

Comment:
huh, I would expect CI to yell at you about adding a migration guide like this one https://docs.airbyte.com/integrations/destinations/snowflake-migrations/

Could you add one for langchain? I'll take a look at why it didn't validate this tomorrow